### DISCUSSION TITLE
Add 300°C all_metal_hotend firmware version

### DISCUSSION THREAD

**Discussion by hendrixradek** [mriscoc/Ender3V2S1#461] [Ideas]
Hi,
 is possible to add firmware version with higher hot end temperature limit?

**Comment by mriscoc** [comment]
Look for C104 custom G-code

**Comment by hendrixradek** [comment]
So only what i need to do is put C104 U1 315 into my slicer in before print section to get max 300C?

**Reply by Across32** [reply]
Working this G-code for you?

**Reply by hendrixradek** [reply]
No it doesn't work. I try slightly increase max temp to 270C so i set C104 U1 285 but while printing max tem still got to 260C.
I put custom g-code in before print gcode section in slicer.

**Reply by MarkforU** [reply]
Have you tried this...
https://github.com/mriscoc/Ender3V2S1/issues/144

**Reply by mriscoc** [reply]
> No it doesn't work. I try slightly increase max temp to 270C so i set C104 U1 285 but while printing max tem still got to 260C. I put custom g-code in before print gcode section in slicer.

You forgot the "T" in the command, it is `C104 U1 T285` <- that will allow you to set the temp up to 285-15 = 270°C. Don't forget a save settings or an M500 afterwards.
